test/graphql/Offer/queries/getApprovedOffers.test.ts

```diff
@@ -107,6 +109,18 @@ describe("getApprovedOffers", () => {
       admin: await AdminGenerator.extension()
     });
     await createOfferWith(ApprovalStatus.rejected, Secretary.graduados, ApplicantType.both);
+    const { uuid } = await createOfferWith(
+      ApprovalStatus.rejected,
+      Secretary.graduados,
+      ApplicantType.both
+    );
+    [, [expiredApprovedByGraduadosAndExtensionBothOffer]] = await Offer.update(
+      { graduatesExpirationDateTime: moment().subtract(1, "days").endOf("day") },
```

Comment:
esto no es equivalente a beginning of day?

Reply:
imagino que esta bien hecho y no. Deberias tener diferencia de 1 min

Reply:
![image](https://user-images.githubusercontent.com/20487262/97438964-544f9f80-1904-11eb-8e78-c9ac90f41dec.png)

1 segundo, pareciera

Reply:
ante la equivalencia en la practica, sugiero usar directamente start of day que restar un dia e ir al final